README.md

```diff
@@ -91,10 +91,10 @@ Thomas' maternal grandfather [James L. Greene](https://twitter.com/mcgonagle/sta
  Boston is a very reserved business culture. Thomas went to a business-focused school for undergraduate and graduate school, and the combination of growing up in Boston and coming up going to Bentley has shaped him. He is most comfortable being formal with customers. With that said, he is certainly capable of creating high quality relationships with customers. Thomas is proud to say that he has become friends with several customers and sees them socially outside of work. 
 
 ## disAbility and Burnout
-Thomas is a disAbility and Burnout awareness activist. He has lead special interest groups at previous employers promoting awareness of disAbility and strives to make every work place as equatable as possible. Thomas co-hosts a peer support talk show on Burnout for the CNCF community on Thursdays at 12PM ET. 
+Thomas is a disAbility and Burnout awareness activist. He has lead special interest groups at previous employers promoting awareness of disAbility and strives to make every work place as equitable as possible. Thomas co-hosts a peer support talk show on Burnout for the CNCF community on Thursdays at 12PM ET. 
 
 ## Thomas' values 
-Thomas' values were heavily influenced by his experience growing up playing sports and participating in the Boy Scouts. Thomas strives to be the very best teammate he can be, and to live by the Scout Law as an adult. 
+Thomas' values were heavily influenced by his experience growing up playing sports and participating in the Boy Scouts. Thomas strives to be the very best teammate he can be, and to live by the [Scout Law](https://www.scouting.org/about/faq/question10/) as an adult. 
```

Comment:
Reading through I suspected a reader less familiar would be asking themselves "what is the Scout Law?" at this point. You list it below with your individualized responses to each point, but I thought it could be helpful to link to the "official" list.